tests/unittests/test_datasource/test_azure.py

```diff
@@ -1613,6 +1617,32 @@ def test_fabric_data_included_in_metadata(self):
         self.assertTrue(ret)
         self.assertEqual('value', dsrc.metadata['test'])
 
+    def test_instance_id_case_insensitive(self):
+        """Return the previous iid when current is a case-insensitive match."""
+        lower_iid = EXAMPLE_UUID
```

Comment:
```suggestion
        lower_iid = EXAMPLE_UUID.lower()
```

in case someone comes through and capitalises it?